packages/flutter/lib/src/services/text_editing_delta.dart

```diff
@@ -24,15 +24,28 @@ TextAffinity? _toTextAffinity(String? affinity) {
   return null;
 }
 
-/// Replaces a range of text in the original string with the text given in the
-/// replacement string.
+// Replaces a range of text in the original string with the text given in the
+// replacement string.
 String _replace(String originalText, String replacementText, int start, int end) {
```

Comment:
You probably don't need this function in first place? https://master-api.flutter.dev/flutter/dart-core/String/replaceRange.html